api/services/sandbox/v1/sandbox.proto

```diff
@@ -95,6 +95,7 @@ service Controller {
 	rpc Status(ControllerStatusRequest) returns (ControllerStatusResponse);
 	rpc Shutdown(ControllerShutdownRequest) returns (ControllerShutdownResponse);
 	rpc Metrics(ControllerMetricsRequest) returns (ControllerMetricsResponse);
+	rpc UpdateResource(ControllerUpdateResourceRequest) returns (ControllerUpdateResourceResponse);
```

Comment:
nit: Just `Update` ?

Reply:
I'm not sure, as the Store already has an `Update` in it, and this API is only for updating resources of a sandbox dynamically before or after we start or delete a task inside the sandbox. `UpdateResource` looks more accurate to describe the function of this API. But I can change the name of API if you still think that is better.